metaverse_components/wear.js

```diff
@@ -178,21 +192,36 @@ export default (app, component) => {
     }
   });
 
-  const _unwear = () => {
+  const _unwear = e => {
     if (wearSpec) {
-      const physicsObjects = app.getPhysicsObjects();
-      for (const physicsObject of physicsObjects) {
-        physicsManager.enableActor(physicsObject);
-      }
-
-      app.scale.copy(initialScale);
-      app.updateMatrixWorld();
-
+      // const physicsObjects = app.getPhysicsObjects();
+      // for (const physicsObject of physicsObjects) {
+      //   physicsManager.enableActor(physicsObject);
+      // }
       wearSpec = null;
       modelBones = null;
     }
   };
 
+  const resettransform = e => {
+    if (e.player) {
+      player = e.player;
+      const avatarHeight = e.player.avatar ? e.player.avatar.height : 0;
+      e.app.position
+        .copy(e.player.position)
+        .add(
+          localVector
+            .set(0, -avatarHeight + 0.5, -1.0)
+            .applyQuaternion(e.player.quaternion),
+        );
+      e.app.quaternion.identity();
+      e.app.scale.set(1, 1, 1);
+      e.app.updateMatrixWorld();
+    }
+  };
+
+  app.addEventListener('resettransform', resettransform);

```

Comment:
Is this still used? We should not emit events in order to "call a method". There is probably a better way to do this.